trs_filer/api/20200404.0aa5443.openapi.yaml

```diff
@@ -575,6 +575,7 @@ components:
       description: Amount of records to return in a given page.
       schema:
         type: integer
+        minimum: 0
```

Comment:
Sorry to be so strict here, but I'd also prefer this one to be removed. While I agree that it _should_ be part of the specs (and lead to a `400/Bad Request` for negative values), as long as it isn't we need to find a workaround. Specifically, I think this param should in the app config and then be read from there in the controller to manually set the value and check that it isn't negative. In case the client _does_ pass a negative value, I think a [`422/Unprocessable Entity`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422) response seems most appropriate to me.

Reply:
We already have a default value of limit as 1000 added [here](https://github.com/elixir-cloud-aai/trs-filer/blob/43896bbb2edf1849396d4f7b382f9d814a12817e/trs_filer/api/20200404.0aa5443.openapi.yaml#L579). Hence, just added a negative value check.